app/assets/javascripts/discourse/app/components/d-button.js

```diff
@@ -98,9 +98,9 @@ export default Component.extend({
         // There is already a warning in the console.
         this.sendAction("action", this.actionParam);
```

Comment:
shouldn't we pass the event here too ?

Reply:
I don't think so, this is deprecated, I don't want to support it anymore.